Sources/StreamChatUI/Utils/Extensions/DifferenceKit+Stream.swift

```diff
@@ -26,12 +26,14 @@ extension UITableView {
     ///                updates should be stopped and performed reloadData. Default is nil.
     ///   - setData: A closure that takes the collection as a parameter.
     ///              The collection should be set to data-source of UITableView.
+    ///   - completion: A completion handler block to execute when all of the operations finish. This block takes a single Boolean parameter that contains the value true if all of the related animations completed successfully or false if they were interrupted.
```

Comment:
@laevandus I think there is no completion handler in DiffKit for a reason, I remember seeing closed issues because of this, example: https://github.com/ra1028/DifferenceKit/issues/45